python/packages/autogen-agentchat/src/autogen_agentchat/teams/_group_chat/_message_store.py

```diff
@@ -0,0 +1,44 @@
+from abc import ABC, abstractmethod
+from typing import List, Optional
+
+from ...messages import BaseChatMessage
+
+
+class MessageStore(ABC):
```

Comment:
It should have `save_state` and `load_state` methods to be used together with the agent classes. 

Also, it should subclass `Component` to enable serializable component config.

Reply:
See example in `autogen_core.mode_context.ChatCompletionContext`.